test/helpers/kubectl.go

```diff
@@ -412,6 +412,57 @@ func (kub *Kubectl) BackgroundReport(commands ...string) (context.CancelFunc, er
 	return cancel, nil
 }
 
+// PprofReport runs pprof on cilium nodes each 5 minutes and saves the data
+// into the test-output.log
```

Comment:
This documentation is inaccurate. Each dump of `gops pprof-cpu 1` is saved into its own file per-pod.